sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/Crashes.java

```diff
@@ -182,6 +197,7 @@ private Crashes() {
         thread.start();
         mHandler = new Handler(thread.getLooper());
         mLastSessionCrashProcessingStatus = PREPARE_PROCESSING;
+        mSemaphore = new Semaphore(0);
```

Comment:
Yes, you are right. It will block forever if there are more than 1 callers. Changed the implementation by using CountDownLatch.